src/utils.js

```diff
@@ -392,3 +392,14 @@ export function combineURLs(baseURL, relativeURL) {
 export const isPushNotificationAvailableForUser = (state) => (
   state.pushNotification.available && state.pushNotification.deviceRegistered
 );
+
+/**
+ * Get Nano Contract feature toggle state from redux.
+ *
+ * @param {Object} state Redux store state
+ *
+ * @returns {boolean} the Nano Contract feature toggle state.
+ */
+export const getNanoContractFeatureToggle = (state) => (
```

Comment:
Should we create a new file for feature flag selectors?

Reply:
Hummm... I think it is nice, but doing it for all selectors. We have a folder for `reducers`, maybe we can create a folder for `selectors`. What do you think?